### ISSUE TITLE
Surrounding on first string

### ISSUE THREAD

**Issue by Riddlerrr** [t9md/atom-vim-mode-plus#940]
Atom 1.21.2, vim-mode-plus 1.15.0, Macos 10.13.

Just a small visual issue.

Type `c s` in the first string of file and don't see that symbol will be changed:
![2017-11-01 14 11 22](https://user-images.githubusercontent.com/517356/32272359-c900cdac-bf0e-11e7-976f-7f5fa0b8166f.png)

expected behavior:
![2017-11-01 14 12 02](https://user-images.githubusercontent.com/517356/32272364-cafa5d1c-bf0e-11e7-82ea-9b9cbbed0c3f.png)

**Comment by t9md** [comment]
Strange, in my environment, I can see hover innboth case.
Cam you reproduce it always?

**Comment by Riddlerrr** [comment]
> Cam you reproduce it always?

Yes, I have tried all standard UI themes and syntax themes. It reproduce in any case.

**Comment by t9md** [comment]
![main_js_ ___github_atom-choose-pane](https://user-images.githubusercontent.com/155205/32310991-78bb94b4-bfd9-11e7-96c6-ff10fdfca4de.jpg)


Really... this is my case, see overlay come front of tab.
I want to fix, but how? want to repro and want to know the cause.

**Comment by Riddlerrr** [comment]
Maybe it depends on the OS?

**Comment by t9md** [comment]
If someone can reproduce issue, put me how-to here please.
